components/compliance-service/dao/pgdb/jobs.go

```diff
@@ -748,41 +748,48 @@ func (db *DB) GetJob(id string) (*jobs.Job, error) {
 func (db *DB) DeleteJob(id string) error {
 	_, err := db.Exec(softDeleteJobById, id)
 	if err != nil {
-		return errors.Wrap(err, "DeleteJob unable to delete job")
+		logrus.Errorf("DeleteJob unable to delete job : %s", err.Error())
+		return errors.New("DeleteJob unable to delete job")
```

Comment:
please check with the product on all the returned error messages in this file. These messages can write better.

Reply:
Updated code